src/service/tatum/tatum.ts

```diff
@@ -9,18 +9,23 @@ import { Notification } from '../notification'
 import { Rates } from '../rate'
 import { LoadBalancer } from '../rpc/generic/LoadBalancer'
 import { Token } from '../token'
-import { WalletProvider } from '../walletProvider'
+import { MetaMask, WalletProvider } from '../walletProvider'
 import { ApiVersion, TatumConfig } from './tatum.dto'
-import { TatumSdkContainer, TatumSdkExtension } from "../extensions";
+import {
+  ExtensionConstructorOrConfig,
+  ITatumSdkContainer,
+  TatumSdkContainer,
+  TatumSdkExtension,
+} from "../extensions";
 
 export interface ITatumSdkChain {
-    extension<T extends TatumSdkExtension>(type: new (tatumSdkContainer: TatumSdkContainer, ...args: unknown[]) => T): T
+    extension<T extends TatumSdkExtension>(type: new (tatumSdkContainer: ITatumSdkContainer, ...args: unknown[]) => T): T
```

Comment:
Again why to add TatumSdk prefix

Reply:
External use, same case why we have TatumSdk main class not just Sdk